lib/services/workflow-api-service.js

```diff
@@ -67,6 +69,9 @@ function workflowApiServiceFactory(
             var context = configuration.context || {};
             if (node) {
                 context = _.defaults(context, { target: node.id });
+                lookupService.nodeIdToProxy(node.id).then(function(proxy) {
+                    context.proxy = proxy;
```

Comment:
I don't think this will work, because `self.createActiveGraph` below will get called before this promise resolves and the callback is executed.